mobly/controllers/android_device_lib/adb.py

```diff
@@ -215,10 +233,7 @@ def instrument(self, package, options=None, runner=None):
         options_string = ' '.join(options_list)
 
         instrumentation_command = 'am instrument -r -w %s %s/%s' % (
-            options_string,
-            package,
-            runner,
-        )
+            options_string, package, runner, )
```

Comment:
This change with the comma at the end seems accidental

Reply:
Done.
Comma has always been there, just linter merged the lines..